azure-kusto-data/azure_bdist_wheel.py

```diff
@@ -7,13 +7,19 @@
 from distutils import log as logger
 import os.path
 
+# review: the 'wheel' package has no officially public API, so be aware that this could break at any time.
```

Comment:
@brettcannon I know, but that's the way we support setuptools and wheel with namespace packages across all SDK. They all have this file.